src/Common/DSSClass.pas

```diff
@@ -553,6 +554,7 @@     TDSSContext = class(TObject)
         DefaultBaseFreq  :Double;
         DaisySize        :Double;
         
+        EventLog:  array of TEventLog;
```

Comment:
We could, but it's too much hassle to clear it from everywhere. I've removed updating it in the `AppendLog/CreateLog` calls, this at least will save some memory and activity.

Also, I tried using TList before, as it would be a bit more convenient, but couldn't make the passing to the C side work. Granted, I haven't spent too much time there though, will check how to do it later.

Reply:
if you can't just send `EventLog.Items` you might be able to use `EventLog.List`

Reply:
Will check later.